tests/unit/test_ncf_singlenode.py

```diff
@@ -0,0 +1,163 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import pytest
+import itertools
+import numpy as np
+import pandas as pd
+import os
+import shutil
+from reco_utils.recommender.ncf.ncf_singlenode import NCF
+from reco_utils.recommender.ncf.dataset import Dataset
+
+N_NEG = 5
+N_NEG_TEST = 10
+
+from reco_utils.common.constants import (
+    DEFAULT_USER_COL,
+    DEFAULT_ITEM_COL,
+    DEFAULT_RATING_COL,
+    DEFAULT_TIMESTAMP_COL,
+)
+
+from tests.ncf_common import python_dataset_ncf, test_specs_ncf
+
+@pytest.mark.parametrize(
```

Comment:
I think here we are missing the gpu test mark: `@pytest.mark.gpu` see this example: https://github.com/Microsoft/Recommenders/blob/staging/tests/unit/test_template_gpu.py. It will be the same in every test that has NCF

Reply:
Thanks, fix it!